event/event.go

```diff
@@ -8,26 +8,35 @@ import (
 	"go.uber.org/zap/zapcore"
 
 	uuid "github.com/satori/go.uuid"
+	"github.com/serverless/event-gateway/internal/zap"
 )
 
 // Event is a default event structure. All data that passes through the Event Gateway is formatted as an Event, based on
 // this schema.
 type Event struct {
-	Type       Type        `json:"event"`
-	ID         string      `json:"id"`
-	ReceivedAt uint64      `json:"receivedAt"`
-	Data       interface{} `json:"data"`
-	DataType   string      `json:"dataType"`
+	Namespace          string `json:"namespace"`
+	EventType          Type   `json:"event-type"`
+	EventTypeVersion   string `json:"event-type-version"`
+	CloudEventsVersion string `json:"cloud-events-version"`
+	Source             struct {
+		SourceType string `json:"source-type"`
+		SourceID   string `json:"source-id"`
+	} `json:"source"`
+	EventID     string                 `json:"event-id"`
+	EventTime   uint64                 `json:"event-time"`
+	ContentType string                 `json:"content-type"`
+	Extensions  zap.MapStringInterface `json:"extensions"`
+	Data        interface{}            `json:"data"`
 }
 
 // New return new instance of Event.
 func New(eventType Type, mime string, payload interface{}) *Event {
 	return &Event{
-		Type:       eventType,
-		ID:         uuid.NewV4().String(),
-		ReceivedAt: uint64(time.Now().UnixNano() / int64(time.Millisecond)),
-		DataType:   mime,
-		Data:       payload,
+		EventType:   eventType,
+		EventID:     uuid.NewV4().String(),
+		EventTime:   uint64(time.Now().UnixNano() / int64(time.Millisecond)),
+		ContentType: mime,
```

Comment:
`New()` should include a value for `cloud-events-version` (`0.1`) and `source` (TBD),